telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/command/autocomplete/AutocompleteHandler.java

```diff
@@ -17,26 +19,36 @@
     private static final Logger log = LoggerFactory.getLogger(AutocompleteHandler.class);
 
     private final BotApi botApi;
+    private final AutocompleteConfig config;
+    private final Map<String,List<String>> autocomplete = new HashMap<String,List<String>>();
 
     @Autowired
-    public AutocompleteHandler(BotApi botApi) {
+    public AutocompleteHandler(BotApi botApi, AutocompleteConfig config) {
         this.botApi = botApi;
+        this.config = config;
+        config.getTemplate().forEach(this::parse);
+    }
+
+    private void parse(String phrase){
+        List<String> temp = new ArrayList<String>();
+        String key = null;
+
+        for (String part: phrase.split(" ")) {
```

Comment:
I had thought about it, but i forgot realize. It will be done in Autocomplete 2.1.